### PR TITLE
Remove read_more

### LINKED ISSUE #6: Avoid read_more
Currently, `Zoo.LANGUAGE` includes `read_more : string -> bool`, which is used in the interactive toplevel to determine whether further lines should be read before the input is parsed. I vaguely remember that I have seen this resolved automatically, though I do not remember how and where. Any ideas?

### PR DESCRIPTION
**Pull request by matijapretnar**
Closes #6.

### CODE REVIEW

File: src/zoo.ml [hunk 1]
```diff
@@ -197,7 +196,8 @@ struct
         | None -> fatal_error "I am sorry but this language has no interactive toplevel."
       in
       Format.printf "%s -- programming languages zoo@\n" L.name ;
-      Format.printf "Type %s to exit@." eof ;
+      Format.printf "Type %s to exit@\n" eof ;
```
Comment (andrejbauer) [inline src/zoo.ml]: `Format.printf "Type %s to exit, split long lines with \\" L.name`

Comment (matijapretnar) [inline src/zoo.ml]: Would it be better to remove the period, because `split long lines with \.` can be confusing?

Comment (andrejbauer) [inline src/zoo.ml]: sure

Comment (andrejbauer) [inline src/zoo.ml]: `Format.printf "Split long lines with \\, type %s to exit.@\n" L.name`

I feel this stuff is just clutter. Can we maybe put it in the usage message?

Comment (matijapretnar) [inline src/zoo.ml]: We could, but it is relevant to the toplevel. What about putting it in the main README?

**Pull request merged**